apps/api/src/app/environments-v1/usecases/output-renderer-usecases.ts

```diff
@@ -0,0 +1,143 @@
+// Base interface for all renderers
+import { render } from '@maily-to/render';
+import {
+  ChatRenderOutput,
+  EmailRenderOutput,
+  InAppRenderOutput,
+  PushRenderOutput,
+  RedirectTargetEnum,
+  SmsRenderOutput,
+  TipTapNode,
+} from '@novu/shared';
+import { z } from 'zod';
+import { ExpendEmailEditorSchemaUseCase } from '../../environments/render/email-schema-extender';
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please fix the typo (`Expend` -> `Extend`). The filename also appears to be different from the class name, this doesn't follow the existing convention. Please rename either the filename or the class name to align on naming.‏

Reply:
there are multiple use-cases here, it cannot be aligned to all...